README.md

```diff
@@ -75,6 +75,23 @@ docId, _ := db.InsertOne("myCollection", doc)
 fmt.Println(docId)
 ```
 
+### Importing and Exporting Collections
+
+CloverDB is capable of easily importing and exporting collections to JSON format regardless of the storage engine used.
+
+```go
+db, _ := c.Open("clover-db")
+
+db.ImportCollection("todos", "test/data/todos.json") // the content of todos.json will be imported in the `todos` collection
+docs, _ := db.Query("todos").FindAll()
+for _, doc := range docs{
+  // you can now work with each document.
+  doc.Set("completed", false)
+}
+
+db.ExportCollection("todos", "test/data/exported.json") // will export the `todos` documents to `exported.json` file, the `completed` field for each document will be `false`
```

Comment:
> will export the `todos` documents to `exported.json` file, the `completed` field for each document will be 
 false.

This statement is not true. You are using  `doc.Set("completed", false)`, so your document object is being updated (in-memory), but you are not saving the updated document on the db.